packages/avatar-group/test/avatar-group-test-styles.js

```diff
@@ -0,0 +1,22 @@
+import { addLumoGlobalStyles } from '@vaadin/vaadin-lumo-styles/global.js';
```

Comment:
Let's use `addGlobalThemeStyles` from ThemableMixin package instead:

```suggestion
import { addGlobalThemeStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
```

The only thing that `addLumoGlobalStyles` does on top of that is `lumo-` prefix which we don't need.

Reply:
Good catch, done.